src/utils/styleTagsFromHtmlExtracter.js

```diff
@@ -0,0 +1,83 @@
+import htmlparser from "htmlparser2"
+
+/**
+ * Extract styles from html markup, returning the exctacted
+ * code and mappings.
+ *
```

Comment:
Solved and added tests for scenario. If no extraction is done then it returns the original code input.